pod/perldelta.pod

```diff
@@ -311,13 +311,13 @@ XXX
 
 =head2 Discontinued Platforms
 
+XXX List any platforms that this version of perl no longer compiles on.
+
 =over 4
 
-=item Symbian
+=item XXX-some-platform
 
-Support code relating to Symbian has been removed.  Symbian was an
-operating system for mobile devices.  The port was last updated in July
-2009, and the platform itself in October 2012.
+XXX
```

Comment:
This shouldn't be removed unless you're reverting the Symbian removal.

Reply:
this is fixed as part of 24168917613aa2d6aa5b43f3c0209d9c8c4125de